index.js

```diff
@@ -196,7 +196,7 @@ module.exports = {
     'no-whitespace-before-property': 'error',
     'object-curly-newline': ['off', {multiline: true}],  // TODO: enable once https://github.com/eslint/eslint/issues/6488 is resolved
```

Comment:
unrelated to pull request but this looks resolved, not sure what option you wanted to set it to - https://github.com/eslint/eslint/issues/6488